src/ngraph/op/experimental/layers/detection_output.hpp

```diff
@@ -74,5 +74,11 @@ namespace ngraph
         private:
             DetectionOutputAttrs m_attrs;
         };
+
+        namespace v1
+        {
+            using op::DetectionOutput;
+            using op::DetectionOutputAttrs;
+        }
```

Comment:
Might be better to move all the layer ops into namespace v1 directly instead of aliasing.